src/sage/schemes/berkovich/berkovich_cp_element.py

```diff
@@ -426,9 +427,10 @@ def _custom_abs(self, x):
 
         ::
 
-            sage: B = Berkovich_Cp_Affine(Qp(3))                                        # optional - sage.rings.padics
-            sage: Q1 = B(9)                                                             # optional - sage.rings.padics
-            sage: Q1._custom_abs(Q1.center())                                           # optional - sage.rings.padics
+            sage:
+            sage: B = Berkovich_Cp_Affine(Qp(3))
```

Comment:
?

Reply:
Perhaps a defect of `sage -fixdoctests`?

Reply:
Yes, this happens when a file tag is present and the block tag is updated. I'll fix this in the fixer.

Reply:
Removed all these empty doctest lines in 12ee3a4f018